Gemfile

```diff
@@ -19,7 +19,7 @@ group :development, :test do
 end
 
 group :development, :lint do
-  gem 'rubocop', '~> 0.90.0'
+  gem 'rubocop', '~> 0.90'
```

Comment:
This won't install Rubocop 1.0+ though, should we take this opportunity to go `~> 1.19`?